README.md

```diff
@@ -20,6 +20,11 @@ The output is uploaded to an Azure storage blob, a suitable connection string sh
 For further details see [Azure Blob Storage](https://azure.microsoft.com/en-gb/services/storage/blobs/).
 
 The ETL retrieves the ODS codes for all Pharmacies from the Syndication API, then visits the organisation API to obtain full pharmacy information.
+An initial list of ODS codes is retrieved from Azure storage. The most recently created file beginning `pharmacy-seed-ids` is used as the source of the data.
```

Comment:
I don't know if the reference to Azure Blob storage could be made more consistent in this text. It varies from including blob to not and the capitalisation.